src/rooms/game.lua

```diff
@@ -183,6 +184,8 @@ function Game:levelComplete(data)
     end
 
     local neighbors = LDtk.get_neighbours(currentLevelName, direction)
+   
+    if not neighbors then return end
```

Comment:
Not sure why you added this line. If there is no neighbour available, "neighbors" is an empty table (which evaluates to "true" in a boolean op).

Let's instead use: "assert(#neighbors > 0, "No neighbors available for: " .. currentLevelName .. " in direction:" .. direction)"

Reply:
the assert was throwing when I created a level with no neighbors for physics test which was not helpful, fwiw